### ISSUE TITLE
stuck on "calculating publication status from GitHub'

### ISSUE THREAD

**Issue by akolatronika** [oleeskild/obsidian-digital-garden#578]
Hi I am stuck on this screen when I click on the plugin on the left hand side of Obsidian. I already checked that my main branch is titled "Main". Is there something else I could fix?
![Screenshot 2024-04-08 130523](https://github.com/oleeskild/obsidian-digital-garden/assets/138243712/d8233d0f-e0cc-4210-9212-83519e7d7333)

**Comment by pstitalia0603** [comment]
this makes me so sad. Happening to me as well!

**Comment by pstitalia0603** [comment]
seems to have sorted itself out.

**Comment by akolatronika** [comment]
yes it seems to have fixed itself on its own, i just double checked. i will close the issue.

**Comment by oleeskild** [comment]
It would be helpful to see what the error message is. Could you open the inspector tools in Obsidian, open the publication center, and see if there are any errors in the Console, and share a screenshot here?

You should be able to bring it up by pressing CMD+Option+I on mac and CTRL+Alt+I on Windows.

**Comment by yulv706** [comment]
Here is the error message when I open the publication center:
plugin:digitalgarden:23795 Uncaught (in promise) URIError: URI malformed
    at decodeURI (<anonymous>)
    at DigitalGardenSiteManager.eval (plugin:digitalgarden:23795:27)
    at Generator.next (<anonymous>)
    at eval (plugin:digitalgarden:90:61)
    at new Promise (<anonymous>)
    at __async (plugin:digitalgarden:74:10)
    at DigitalGardenSiteManager.getImageHashes (plugin:digitalgarden:23787:12)
    at PublishStatusManager.eval (plugin:digitalgarden:23328:56)
    at Generator.next (<anonymous>)
    at fulfilled (plugin:digitalgarden:77:24)

![Image](https://github.com/user-attachments/assets/ebb92877-ae68-4a66-8c4b-3740b95b2910)

**Comment by yulv706** [comment]
**Sorry, there was a problem with the previous copy, but the following is complete(probably)**

```txt
Uncaught (in promise) URIError: URI malformed at decodeURI (<anonymous>) at DigitalGardenSiteManager.eval (plugin:digitalgarden:23795:27) at Generator.next (<anonymous>) at eval (plugin:digitalgarden:90:61) at new Promise (<anonymous>) at __async (plugin:digitalgarden:74:10) at DigitalGardenSiteManager.getImageHashes (plugin:digitalgarden:23787:12) at PublishStatusManager.eval (plugin:digitalgarden:23328:56) at Generator.next (<anonymous>) at fulfilled (plugin:digitalgarden:77:24)
 eval @ plugin:digitalgarden:23795 
 eval @ plugin:digitalgarden:90 
 __async @ plugin:digitalgarden:74 
 getImageHashes @ plugin:digitalgarden:23787 
 eval @ plugin:digitalgarden:23328 
 fulfilled @ plugin:digitalgarden:77 
 Promise.then   
 step @ plugin:digitalgarden:19877 
 eval @ plugin:digitalgarden:19879 
 __awaiter @ plugin:digitalgarden:19861 
 getPublishStatus @ plugin:digitalgarden:22419 
 run @ plugin:digitalgarden:19045 
 eval @ plugin:digitalgarden:19513 
 flush @ plugin:digitalgarden:19395 
 init @ plugin:digitalgarden:19594 
 PublicationCenter @ plugin:digitalgarden:22583 
 modal.onOpen @ plugin:digitalgarden:23260 
 (anonymous) @ app.js:1 
 PublicationCenter2.open @ plugin:digitalgarden:23272 
 openPublishModal @ plugin:digitalgarden:29586 
 eval @ plugin:digitalgarden:29269 
 eval @ plugin:digitalgarden:90 
 __async @ plugin:digitalgarden:74 
 eval @ plugin:digitalgarden:29268 
```

![Image](https://github.com/user-attachments/assets/bf5dd8a4-1fa3-453b-8abc-560360863a38)

**Comment by oleeskild** [comment]
Thanks! Think I've found your issue. Try updating to the newest version of the plugin in obsidian (2.58.1) and see if that fixes your issue. Let me know if it worked out or not.

**Comment by yulv706** [comment]
I can't tell you how excited I am. This really solved my problem. Thank you very much

![Image](https://github.com/user-attachments/assets/ef3e1f5d-0690-4f55-94a2-349686688b65)

**Comment by oleeskild** [comment]
Great! Thank for posting the errors. I'm sure a lot of other people experienced this bug as well 🙌
